src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+
+  const block = document.createElement('div');
+
+  block.setAttribute('class', `notification ${type}`);
```

Comment:
Use `classList.add()` instead of `setAttribute()` for adding classes to the element.